tests/actions.go

```diff
@@ -92,9 +96,14 @@ func NewOperatorActions(cli versioned.Interface,
 		asCli:       asCli,
 		tcStsGetter: kubeCli.AppsV1(),
 		// tcStsGetter:  helper.NewHijackClient(kubeCli, asCli).AppsV1(),
-		tidbControl:  controller.NewDefaultTiDBControl(),
 		pollInterval: pollInterval,
 		cfg:          cfg,
+		fw:           fw,
+	}
+	if fw != nil {
+		oa.tidbControl = proxiedtidbclient.NewProxiedTiDBClient(fw)
+	} else {
+		oa.tidbControl = controller.NewDefaultTiDBControl()
```

Comment:
keep backward compatibility if `fw` is not provided. we can remove this when we can run stability test cases outside of the Kubernetes cluster.